### PR TITLE
Footer update (new version)

### PR DESCRIPTION
**Pull request by prycrane**
- Textual updates for the footer (@LPipes4)
- Config file update (@delisma )

Preview

- https://deploy-preview-160--systeme-conception-canada-ca.netlify.app/configurations-conception-communes/pied-page.html
- https://deploy-preview-160--systeme-conception-canada-ca.netlify.app/configurations-conception-communes/pied-page-contextuelle.html
- https://deploy-preview-160--systeme-conception-canada-ca.netlify.app/configurations-conception-communes/pied-page-principale.html
- https://deploy-preview-160--systeme-conception-canada-ca.netlify.app/configurations-conception-communes/pied-page-sous.html

English pull request: https://github.com/canada-ca/design-system/pull/267

__________________________________________________________________

This pull request replaces [fix: site-footer-edits-fr #131](https://github.com/canada-ca/systeme-conception/pull/131) (No longer valid)

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -50,3 +52,26 @@ defaults:
...
+      breadcrumbs:
+      - title: À propos de Canada.ca
+        link: https://www.canada.ca/fr/gouvernement/a-propos.html
+      - title: Système de conception de Canada.ca
+        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html
+      - title: Bibliothèque de modèles et de configurations de conception
+        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception/bibliotheque-modeles.html
+      - title: Pied de page général
+        link: https://conception.canada.ca/configurations-conception-communes/pied-page.html
+      permalink: /configurations-conception-communes/:basename:output_ext
```
Comment (delisma) [inline _config.yml:77]:
```suggestion
  - scope:
      path: "configurations-conception-communes/pied-page-general/*.md"
    values:
      breadcrumbs:
      - title: À propos de Canada.ca
        link: https://www.canada.ca/fr/gouvernement/a-propos.html
      - title: Système de conception de Canada.ca
        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html
      - title: Bibliothèque de modèles et de configurations de conception
        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception/bibliotheque-modeles.html
      - title: Pied de page général
        link: https://conception.canada.ca/configurations-conception-communes/pied-page.html
      permalink: /configurations-conception-communes/:basename:output_ext
```
Let's keep the change to the config files to the bare minimum. The master branch will already have the previous mods by the time we merge this one.

File: _layouts/default.html
Comment (delisma) [inline _layouts/default.html:1]: Let's keep the layouts file out of this PR. The master branch will already have these mods by the time we merge this one.

File: configurations-conception-communes/pied-page-general/pied-page-contextuelle.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
+---
+altLangPage: "https://design.canada.ca/common-design-patterns/site-footer-contextual.html"
+date: 2022-11-30
```
Comment (delisma) [inline configurations-conception-communes/pied-page-general/pied-page-contextuelle.md:3]: Note: We should think of a different approach to keep the history in the frontmatter

File: configurations-conception-communes/pied-page-general/pied-page-contextuelle.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+dateModified: 2023-04-06
+description: "Il s’agit d’un élément facultatif du pied de page général (à l’échelle du site)."
+title: "Pied de page général&nbsp;: Bande contextuelle"
+---
+<p><strong>Dernière mise à jour</strong>&nbsp;:&nbsp;2023-04-06</p>
```
Comment (delisma) [inline configurations-conception-communes/pied-page-general/pied-page-contextuelle.md:8]:
```suggestion
<p><strong>Dernière mise à jour</strong>&nbsp;:&nbsp;{{ page.dateModified }}</p>
```

File: configurations-conception-communes/pied-page-general/pied-page-principale.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
+---
+date: 2022-11-30
+dateModified: 2023-04-06
+description: "Il s’agit d’un élément du pied de page général (à l’échelle du site) qui est obligatoire dans les pages standard."
+title: "Pied de page général&nbsp;: Bande principale"
+---
+<p><strong>Dernière mise à jour</strong>&nbsp;:&nbsp;2023-04-06</p>
```
Comment (delisma) [inline configurations-conception-communes/pied-page-general/pied-page-principale.md:7]:
```suggestion
<p><strong>Dernière mise à jour</strong>&nbsp;:&nbsp;{{ page.dateModified }}</p>
```

File: configurations-conception-communes/pied-page-general/pied-page-sous.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
+---
+altLangPage: "https://design.canada.ca/common-design-patterns/site-footer-sub.html"
+date: 2022-11-30
+dateModified: 2023-04-06
+description: "Il s’agit d’un élément obligatoire du pied de page général (à l’échelle du site)."
+title: "Pied de page général&nbsp;: Bande sous pied de page"
+---
+<p><strong>Dernière mise à jour</strong>&nbsp;:&nbsp;2023-04-06</p>
```
Comment (delisma) [inline configurations-conception-communes/pied-page-general/pied-page-sous.md:8]:
```suggestion
<p><strong>Dernière mise à jour</strong>&nbsp;:&nbsp;{{ page.dateModified }}</p>
```

File: configurations-conception-communes/pied-page.md [hunk 6]
```diff
@@ -0,0 +1,333 @@
+---
+altLangPage: "https://design.canada.ca/common-design-patterns/site-footer.html"
+date: 2019-01-08
+dateModified: 2023-04-06
+description: "Le pied de page général (à l’échelle du site) qui se trouve au bas de chaque page. Il comprend la bande principale, une bande contextuelle et une bande sous pied de page. Il s’agit d’un modèle obligatoire avec des éléments facultatifs."
+title: "Pied de page général"
+---
+<p><strong>Dernière mise à jour</strong>&nbsp; 2023-04-06</p>
```
Comment (delisma) [inline configurations-conception-communes/pied-page.md:8]:
```suggestion
<p><strong>Dernière mise à jour</strong>&nbsp; {{ page.dateModified }}</p>
```

File: configurations-conception-communes/pied-page-general/pied-page-contextuelle.md (hunk 2 above)
Comment (delisma) [reply configurations-conception-communes/pied-page-general/pied-page-contextuelle.md:3]: Maybe a "Latest changes" Jekyll component

**Review by delisma — CHANGES_REQUESTED** [review]

**Comment by delisma** [comment]
This PR should close #131

File: _config.yml (hunk 1 above)
Comment (prycrane) [reply _config.yml:90]: Okay

File: configurations-conception-communes/pied-page-general/pied-page-contextuelle.md (hunk 3 above)
Comment (prycrane) [reply configurations-conception-communes/pied-page-general/pied-page-contextuelle.md]: Done

File: configurations-conception-communes/pied-page-general/pied-page-principale.md (hunk 4 above)
Comment (prycrane) [reply configurations-conception-communes/pied-page-general/pied-page-principale.md]: Done

File: configurations-conception-communes/pied-page.md (hunk 6 above)
Comment (prycrane) [reply configurations-conception-communes/pied-page.md]: Done

File: configurations-conception-communes/pied-page-general/pied-page-sous.md (hunk 5 above)
Comment (prycrane) [reply configurations-conception-communes/pied-page-general/pied-page-sous.md]: Done

File: configurations-conception-communes/pied-page-general/pied-page-contextuelle.md (hunk 2 above)
Comment (prycrane) [reply configurations-conception-communes/pied-page-general/pied-page-contextuelle.md:3]: What history?  Latest changes would be would be dateModified.  Historical changes should be part of a CMS?

File: _layouts/default.html
Comment (prycrane) [reply _layouts/default.html:1]: Indeed.  I just want the previews to work.

**Pull request closed without merging**